.github/workflows/backend.yml

```diff
@@ -1,16 +1,11 @@
 name: Backend
 
 on:
-  pull_request:
   push:
     branches:
-      - '**'
-    paths:
-    - '**.clj*'
```

Comment:
We could save these paths to be a bit more efficient

Reply:
I don't think it's really worth it, since we always need to get the paths correct, including build script, config, etc (missing it will be disastrous) and the cognitive load isn't worth it (just look how long the paths become already).
If later we discover that brute-force like this is causing some pain, we can always come back and optimize it.